src/components/Notifications/Notifications.tsx

```diff
@@ -733,6 +734,9 @@ class NotificationEntry extends React.Component<{notification}, any> {
 
     renderNotification() {
         const notification = this.props.notification;
+        let outcome: any;
+        let now: number;
```

Comment:
TypeScript newb here, but is this technically correct? Or should it be something like `number | undefined` (and same with below)?

Reply:
Typescript understands that it's initially undefined. With member fields of a class, you have to be more specific if you want them to be able to be undefined (otherwise it expects them to be defined within the constructor), but for locally scoped variables such as these, they're undefined if you don't define them, simple enough.